### ISSUE TITLE
paper-input-container , screen reader issue

### ISSUE THREAD

**Issue by uxdesignz** [PolymerElements/paper-input#710]
Actual Result:
Form Labels: The input form field label element is referenced by the form input using aria-labelledby but it has visibility:hidden applied to it after text is entered into the form field. Screen readers may not announce the form field label.

Expected Results
Remove aria-hidden="true" from the <label> element for the form input as this may block some screen readers from announcing the form label.

**Comment by uxdesignz** [comment]
Any update on this?
